qualtran/bloqs/hamiltonian_simulation/hamiltonian_simulation_by_gqsp_test.py

```diff
@@ -99,4 +100,7 @@ def test_hamiltonian_simulation_by_gqsp(
 
 def test_hamiltonian_simulation_by_gqsp_t_complexity():
     hubbard_time_evolution_by_gqsp = _hubbard_time_evolution_by_gqsp.make()
-    _ = hubbard_time_evolution_by_gqsp.t_complexity()
+    t_comp = hubbard_time_evolution_by_gqsp.t_complexity()
```

Comment:
Is there a reason you just picked this test?

Reply:
somewhat arbitrary -- seemed like a reasonably complicated example